README.md

```diff
@@ -18,90 +28,66 @@ One of the most important aspects of collaboration is knowing who else you're wo
-
-
-## Getting started
-
-Use the following snippets to quickly get up and running. More detailed [usage instructions](/docs/usage.md) are available, as are [class definitions](/docs/class-definitions.md) and information on [channel behavior](/docs/channel-behaviors.md).
-
-You can also view a [live demo](https://space.ably.dev) which uses an example slideshow to demonstrate updating an avatar stack, displaying user locations and live cursors.
+* more detailed [usage instructions](/docs/usage.md)
+* [class definitions](/docs/class-definitions.md)  
+* how the Spaces SDK uses [Ably internally](/docs/channel-behaviors.md)
 
 ### Prerequisites
 
 To begin, you will need the following:
 
 * An Ably account. You can [sign up](https://ably.com/signup) for free.
 * An Ably API key. You can create API keys in an app within your [Ably account](https://ably.com/dashboard).
-  * The API key needs the following [capabilities](https://ably.com/docs/realtime/authentication#capabilities-explained): `publish`, `subscribe`, `presence` and `history`.
+  * The API key needs the following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence` and `history`.
 
-You can use [basic authentication](https://ably.com/docs/realtime/authentication#basic-authentication) for testing purposes, however it is strongly recommended that you use [token authentication](https://ably.com/docs/realtime/authentication#token-authentication) in any production environments.
+You can use [basic authentication](https://ably.com/docs/auth/basic) for testing purposes, however it is strongly recommended that you use [token authentication](https://ably.com/docs/auth/token) in production environments.
 
 ### Authenticate and instantiate
 
-Install the Ably JavaScript SDK and the Collaborative Spaces SDK:
+Install the Collaborative Spaces SDK:
 
 ```sh
 npm install ably @ably-labs/spaces
 ```
 
-Import the SDKs and then instantiate the Collaborative Spaces SDK with your Ably API key and pass a [clientId](https://ably.com/docs/realtime/authentication?lang=javascript#identified-clients) (if prototyping, you can use a package like [nanoid](https://www.npmjs.com/package/nanoid) to generate an id):
+To instantiate the Spaces SDK, create an [Ably client](https://ably.com/docs/getting-started/setup) and pass it into the Spaces constructor:
 
 ```ts
 import Spaces from '@ably-labs/spaces';
+import { Realtime } from 'ably';
 
-const spaces = new Spaces({ key: ABLY_API_KEY, clientId: "id" });
+const client = new Realtime.Promise(options);
```

Comment:
Now that's the only way we suggest instantiating, I think we need to spell out options (which will be `{ key: ABLY_API_KEY, clientId: "id" }` in its simplest case)